app/controllers/PaymentError.scala

```diff
@@ -0,0 +1,15 @@
+package controllers
+
+sealed trait PaymentError {
+  def message: String
+}
+
+object PaymentError {
+  val allErrors: Map[String, PaymentError] = Map(PaypalError.toString -> PaypalError)
+
+  def fromString(errorCode: String): Option[PaymentError] = allErrors.get(errorCode)
+}
+
+case object PaypalError extends PaymentError {
+  override def message = "Your contribution using PayPal could not be processed. Please try again or use pay by card."
```

Comment:
I thought "use pay by card" it was strange as well, but that's the text I've been given. 
I would have written "use another payment method" but maybe that's not specific enough. But even if we keep this line it might me more correct to say "use 'contribute with card'" to match the label in the button